impeller/geometry/path_builder.cc

```diff
@@ -38,7 +38,7 @@ PathBuilder& PathBuilder::MoveTo(Point point, bool relative) {
 }
 
 PathBuilder& PathBuilder::Close() {
-  LineTo(subpath_start_);
+  // LineTo(subpath_start_);
```

Comment:
This is also the cause of https://github.com/flutter/flutter/issues/146648#issuecomment-2059881763 .